### PR TITLE
Create MichelbergerHotel.com.xml

### PR DESCRIPTION
**Pull request by pipboy96**

### CODE REVIEW

**Comment by pipboy96** [comment]
Closes #15210.

File: src/chrome/content/rules/MichelbergerHotel.com.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!--
+	Self-signed:
+		- mail
+		- vpn
+-->
```
Comment (J0WI) [inline src/chrome/content/rules/MichelbergerHotel.com.xml]: Wildcard DNS record

Comment (pipboy96) [reply src/chrome/content/rules/MichelbergerHotel.com.xml]: @J0WI I'm not sure what to do here. I guess I should simply add a note in the comment and only include the domains that actually have content (not 404).

Comment (pipboy96) [reply src/chrome/content/rules/MichelbergerHotel.com.xml]: `remote` sends the same response as the non-existent domains, so I removed it.

Comment (J0WI) [reply src/chrome/content/rules/MichelbergerHotel.com.xml]: What about the hosts in https://github.com/EFForg/https-everywhere/pull/17592#pullrequestreview-215209963?

Comment (pipboy96) [reply src/chrome/content/rules/MichelbergerHotel.com.xml]: `shop`: no response at `:443`.
`store`: 404 on both http and https, can't find working url.

Comment (J0WI) [reply src/chrome/content/rules/MichelbergerHotel.com.xml]: Try with `http://shop.michelbergermonkey.com/germany/` or `http://store.michelbergermonkey.com/germany/`

Comment (pipboy96) [reply src/chrome/content/rules/MichelbergerHotel.com.xml]: @J0WI Redirect to HTTP.

Comment (J0WI) [reply src/chrome/content/rules/MichelbergerHotel.com.xml]: Could you add them to the list?

Comment (pipboy96) [reply src/chrome/content/rules/MichelbergerHotel.com.xml]: @J0WI Done.

Comment (pipboy96) [reply src/chrome/content/rules/MichelbergerHotel.com.xml]: @J0WI Done.

Comment (J0WI) [reply src/chrome/content/rules/MichelbergerHotel.com.xml]: Thanks!

**Pull request merged**